src/renderer/components/select/select.tsx

```diff
@@ -68,7 +69,7 @@ export class Select extends React.Component<SelectProps> {
     return this.options.find(opt => opt === value || opt.value === value);
   }
 
-  @computed get options(): SelectOption[] {
+  @computed get options(): readonly (OptionTypeBase | GroupTypeBase<OptionTypeBase>)[] {
```

Comment:
Why do we use here `OptionTypeBase` instead of `OptionType` when:

```
export interface OptionTypeBase {
    [key: string]: any;
}
```

Every `option` must contain at least `value` and `label` fields.